### PR TITLE
Add staging remerge

### PR DESCRIPTION
**Pull request by Olson3R**
Add ability to restage PRs that have been deployed when creating a new staging branch.

### CODE REVIEW

File: lib/octopolo/deployed_branch_merger.rb [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    end
+
+    def remerge_dialog
+      cli.say('Deployed PRs:')
+      cli.say(deployed_prs.map { |pr| "[#{pr.number}] #{pr.title} - #{pr.html_url}" }.join("\n"))
+      cli.highline.choose do |menu|
+        menu.prompt = 'Would you like to re-merge deployed PRs?'
+        menu.choice(:none) { @remerge_pr_numbers = [] }
+        menu.choice(:all) { merge_all }
+        # menu.choice(:choose) {  }
```
Comment (Olson3R) [inline lib/octopolo/deployed_branch_merger.rb]: TODO: Allow the user to choose individual PRs to merge.

File: lib/octopolo/deployed_branch_merger.rb [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      end
+    end
+
+    def merge_all
+      @remerge_pr_numbers = deployed_prs.map(&:number)
+    end
+
+    def can_remerge_branches?
```
Comment (bbergstrom) [inline lib/octopolo/deployed_branch_merger.rb]: Should we allow this for the deployable branch as well?

Comment (Olson3R) [reply lib/octopolo/deployed_branch_merger.rb]: Probably.

File: lib/octopolo/deployed_branch_merger.rb [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+      if should_remerge_deployed_prs
+        merge_all
+      else
+        remerge_dialog
+      end
+
+      return if remerge_pr_numbers.empty?
+      remerge_pr_numbers.each do |pr_number|
+        Octopolo::PullRequestMerger.new(branch_type, pr_number).perform
```
Comment (Olson3R) [inline lib/octopolo/deployed_branch_merger.rb]: Probably should add some notification if there is a merge conflict or some other failure.

**Pull request closed without merging**